### PR TITLE
fixes #1063

### LINKED ISSUE #1063: [enhancement] Go to last edit location
Please *thumbs-up* 👍 this issue if it personally affects you! You can do this by clicking on the emoji-face on the top right of this post. Issues with more thumbs-up will be prioritized.

-----

### What did you do?
`g;`
### What did you expect to happen?
`go to last edit location`
### What happened instead?
nothing
### Technical details:

* VSCode Version: 1.8.0-insider
* VsCodeVim Version: 0.42
* OS: debian/osx

### PR DESCRIPTION
**Pull request by xconverge**
implements g; to go to last edited location

### CODE REVIEW

File: src/history/historyTracker.ts [hunk 1]
```diff
@@ -518,6 +518,26 @@ export class HistoryTracker {
...
   }
 
+  getLastContentChangePosition(): Position[] | undefined {
+    if (this.currentHistoryStepIndex === 0) {
+      return undefined;
+    }
+
+    let prevPos = this.getLastHistoryEndPosition();
+
+    for (var steps of this.historySteps) {
```
Comment (johnfn) [inline src/history/historyTracker.ts]: Can you use `const` here?

File: src/actions/actions.ts [hunk 2]
```diff
@@ -2794,6 +2794,22 @@ class CommandGoToDefinition extends BaseCommand {
   }
 }
 
+@RegisterAction
+class CommandGoToLastEditLocation extends BaseCommand {
+  modes = [ModeName.Normal];
+  keys = ["g", ";"];
+
+  public async exec(position: Position, vimState: VimState): Promise<VimState> {
+    let prevPos = vimState.historyTracker.getLastContentChangePosition();
```
Comment (johnfn) [inline src/actions/actions.ts]: `const` here too.

**Comment by xconverge** [comment]
I want to loop backwards as well instead.

**Comment by xconverge** [comment]
Actually g; and g, are supposed to move through the change list (forward and backward) not just to the last one, so this needs some more attention for sure

**Comment by xconverge** [comment]
Ok @johnfn now this works better now and closer to vims behavior

**Comment by johnfn** [comment]
Very cool.

**Pull request merged**